extension/main.js

```diff
@@ -1,6 +1,11 @@
 (function () {
 	'use strict';
 
+	function createNotificationId() {
+		var id = Math.floor(Math.random() * 9007199254740992) + 1;
```

Comment:
The notification ID should be unique if you want to clear the previous notification before.
In the example given by Google (https://github.com/GoogleChrome/chrome-app-samples/tree/master/samples/gcm-notifications), a new notification is created for each new message receive.
In our case, there is only one "source" of notification : The user account. I think it's not necesseray to create a new ID each time. On the contrary.